modules/ROOT/pages/bundling-skins.adoc

```diff
@@ -34,31 +34,31 @@ IMPORTANT: The handling of content CSS files (such as `+content.css+` or `+conte
 
 [source,js]
 ----
-import contentUiCss from 'tinymce/skins/ui/example/content.css';
+import contentUiSkin from 'tinymce/skins/ui/example/content.css';
```

Comment:
Why was this changed? This isn't related to models and doesn't seem correct as it is a CSS file... if it is then the inline variable below should also be changed.

- [ ] Revert these changes please

Reply:
I should add that `Ui` and `skin` both mean the same thing here so this new name basically doubles up. At the very least, if you don't want to revert this should still include `css` in the name since that's what it's holding.

Reply:
I was already changing this nearly-identical file a bunch of times and noticed they weren't actually identical. At least when they're properly copy/paste they can be updated easily with a search and replace.

The folder location includes both ui and skin, so I figured the variable name could too 🤷 the point I was trying to hint at to customers is part of the skin which isn't optional, whereas regular content CSS is. I guess I'll make it `contentUiSkinCss`.

Reply:
Fair point about the `ui/skin` path, so `contentUiSkinCss` sounds fine.

Re the differences, looks like when that was done someone accidentally swapped the order previously.